app/scripts/controllers/tools.js

```diff
@@ -347,14 +347,18 @@ angular.module('oncokbApp')
             };
             $scope.generateEvidences = function () {
                 $scope.loadingReviewed = true;
+                $scope.displayReviewedData = false;
+                $scope.reviewedData[$scope.data.evidenceType].body = [];
 
                 DatabaseConnector.getReviewedData($scope.reviewedData[$scope.data.evidenceType].evidenceTypes).then(function(response) {
                     if ($scope.data.evidenceType === 'geneSummary' || $scope.data.evidenceType === 'geneBackground') {
                         // key = 'summary' or key = 'background'
                         var key = $scope.reviewedData[$scope.data.evidenceType].keys[1];
                         _.each(response.data, function(item) {
                             var tempObj = {
```

Comment:
Let's create a method to construct the `tempObj`, and give it a proper name. By doing so, you could avoid some code duplication.

Reply:
Created ReviewedData() in VariantFactory.js